responsibleai_text/responsibleai_text/managers/error_analysis_manager.py

```diff
@@ -105,7 +105,7 @@ def predict(self, X):
         predictions = self.predictions[index]
         if self.task_type == ModelTask.MULTILABEL_TEXT_CLASSIFICATION:
             return predictions
-        if self.classes is not None:
+        if self.classes is not None and isinstance(predictions[0], int):
```

Comment:
in the description it mentioned "For error_analysis_manager, in the case where the prediction value is a **string**" why it is int here?

Reply:
Sorry for the confusion! 
The funtionality of line 109 is, it converts predictions from int to string, for example, for binary classification, the predictions is `[0, 1, 0]`, and I convert it to strings at line 109, which will become `['POSITIVE', 'NEGATIVE', 'POSITIVE']`, where self.classes is `['POSITIVE', 'NEGATIVE']`.
And for type of string, we skip line 109, we don't need to go into this if condition at line 109 since it is already type of string, meaning it's already `['POSITIVE', 'NEGATIVE', 'POSITIVE']`